src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.tsx

```diff
@@ -395,3 +341,75 @@ const ButtonHolder = styled('div')`
     margin-left: ${space(0.5)};
   }
 `;
+
+type ActionButtonsProps = {
+  org: Organization;
+  app: SentryApp;
+
+  showDashboard: boolean;
+  showPublish: boolean;
+  showDelete: boolean;
+  onPublish?: () => void;
+  onDelete?: (app: SentryApp) => void;
+  // If you want to disable the publish or delete buttons, pass in a reason to display to the user in a tooltip
+  disablePublishReason?: string;
+  disableDeleteReason?: string;
+};
+const ActionButtons = ({
+  org,
+  app,
+  showDashboard,
+  showPublish,
+  showDelete,
+  onPublish,
+  onDelete,
+  disablePublishReason,
+  disableDeleteReason,
+}: ActionButtonsProps) => {
+  const appDashboardButton = (
+    <StyledButton
+      size="small"
+      icon="icon-stats"
+      to={`/settings/${org.slug}/developer-settings/${app.slug}/dashboard`}
```

Comment:
Our urls should all have a trailing slash

Reply:
@billyvg does it matter if it has a trailing slash in `routes.jsx`? I just tested and it seems the url works either way (it does not have the trailing slash in routes.jsx)

Reply:
@nolachen It doesn't matter in `routes` because it will still match (we should still add it for consistency sake) - I believe we also redirect or rewrite the url to add the trailing slash